packages/gatsby/src/commands/types.ts

```diff
@@ -3,8 +3,8 @@ import { Store, AnyAction } from "redux"
 import { IGatsbyState } from "../redux/types"
 
 export interface ICert {
-  key: Buffer
-  cert: Buffer
+  key: string
+  cert: string
```

Comment:
we are passing ICert Object to child process, so this has to be serializable. Buffer can be serialized but but this was fastest way to fix as Strings are also supported.

Without this I was getting:
```
The "options.cert" property must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Object
```